### PR TITLE
Moving workshops to topics

### PR DESCRIPTION
**Pull request by Lisa1400**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Lisa1400** [comment]
Screenshot from 2021-03-12 10-20-03

**Comment by Lisa1400** [comment]
Screenshot from 2021-03-12 10-23-19

File: content/syllabuses/data-sci.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ _db_id: 2
...
+- {{% contentlink path="topics/general-coding-best-practices" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
 - {{% contentlink path="workshops/survey-design" %}}
-- {{% contentlink path="workshops/google-forms-like-a-boss" %}}
+- {{% contentlink path="topics/google-forms-like-a-boss" %}}
 - {{% contentlink path="projects/morse-code" flavour="python" %}}
 - {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
-- {{% contentlink path="workshops/intro-to-data-ethics" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-sci.md:22]: you can revert this change, the data guys say this needs to be a workshop for teh time being

File: content/topics/intro-to-css/_index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+content_type: topic
+ready: true
+title: Introduction to CSS
+---
+
+> There are two great days in a person's life
+>
+> - the day they are born and the day they discover why.
+
+Topics to be discussed in this workshop can be found here {{% contentlink path="topics/web-frontend/intro-to-css" %}}
```
Comment (ry-oc) [inline content/topics/intro-to-css/_index.md]: is this topic already in the syllabus? If yes this will just become a weird duplicate as it will point to content that is already present.

Maybe move it to depreciated

File: content/topics/intro-to-data-ethics/_index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (ry-oc) [inline content/topics/intro-to-data-ethics/_index.md]: Please revert this as per the comment on the data syllabus.

File: content/syllabuses/data-eng.md [hunk 4]
```diff
@@ -20,13 +20,13 @@ _db_id: 4
...
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python"%}}
-- {{% contentlink path="workshops/big-picture-web-dev" %}}
+- {{% contentlink path="topics/big-picture-web-dev" %}}
 - NCIT 10: Workshop: Giving and receiving constructive feedback
 - NCIT 06: Web design for business
 - NCIT 07: Building an online business
 - NCIT 12: Business and technology
 - NCIT 09: Ethics and Professionalism in IT
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/general-coding-best-practices" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-eng.md]: is content/topics/clean-code/general/_index.md sufficient for clean code? If it is then this can be depreciated

File: content/topics/git-basics/_index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+_db_id: 8
```
Comment (ry-oc) [inline content/topics/git-basics/_index.md]: is content/topics/git/_index.md sufficient? If it is then this can be depreciated

File: content/topics/git-for-teamwork/_index.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+_db_id: 15
```
Comment (ry-oc) [inline content/topics/git-for-teamwork/_index.md]: this can be depreciated as they get  to know how to work together with git in all their projects now

File: content/syllabuses/data-eng.md (hunk 4 above)
Comment (ry-oc) [inline content/syllabuses/data-eng.md]: Is content/topics/clean-code/general/_index.md not already in the syllabus? Does it need to be upgraded to add any missing content covered by the workshop?

Best thing to do is upgrade the topic and depreciate the workshop instead of having them go through both.

Comment (ry-oc) [inline content/syllabuses/data-eng.md]: can content/topics/clean-code/general/_index.md be updated to cover anything this covers?

Then this can be depreciated

File: content/syllabuses/data-eng.md [hunk 7]
```diff
@@ -78,7 +78,7 @@ _db_id: 4
 - {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
 - NCIT 16: Thinking like a machine / Boolean algebra, logic
 - NCIT 17: Search and sort techniques
-- {{% contentlink path="workshops/git-for-teamwork/" %}}
+- {{% contentlink path="topics/git-for-teamwork/" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-eng.md]: This is covered in their practical work and can be depreciated

File: content/syllabuses/data-sci.md [hunk 8]
```diff
@@ -11,7 +11,7 @@ _db_id: 2
 - {{% contentlink path="agile/agile-scrum" %}}
 - NCIT 18: Agile project management
 - NCIT 19: Agile meetings
-- {{% contentlink path="workshops/how-to-be-a-professional" %}}
+- {{% contentlink path="topics/how-to-be-a-professional" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-sci.md]: Can this be changed to a topic that covers the relevent info instead of an outdated presentation?

File: content/syllabuses/data-sci.md (hunk 1 above)
Comment (ry-oc) [inline content/syllabuses/data-sci.md:22]: Data dept says that this needs to stay as a workshop for the time being

File: content/syllabuses/data-sci.md [hunk 9]
```diff
@@ -23,15 +23,15 @@ _db_id: 2
...
 - NCIT 12: Business and technology
 - NCIT 09: Ethics and Professionalism in IT
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/general-coding-best-practices" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
 - {{% contentlink path="workshops/survey-design" %}}
-- {{% contentlink path="workshops/google-forms-like-a-boss" %}}
+- {{% contentlink path="topics/google-forms-like-a-boss" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-sci.md:19]: Are you able to create a topic out of this? If not this change is fine and we can allocate the task to someone else

File: content/syllabuses/data-sci.md [hunk 10]
```diff
@@ -23,15 +23,15 @@ _db_id: 2
 - NCIT 07: Building an online business
 - NCIT 12: Business and technology
 - NCIT 09: Ethics and Professionalism in IT
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/general-coding-best-practices" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
 - {{% contentlink path="workshops/survey-design" %}}
```
Comment (ry-oc) [inline content/syllabuses/data-sci.md]: You skipped this workshop, was this intentional?

File: content/topics/express-js/_index.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+---
+_db_id: 5
+content_type: topic
+ready: true
+title: Express JS
+---
+
+<iframe src="https://drive.google.com/file/d/14RIS-T1WNdpHX7Zu4zNAylRS1_Q3ahZm/preview" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true" style="display:block; margin: 0 auto;"></iframe>
```
Comment (ry-oc) [inline content/topics/express-js/_index.md]: Can http://syllabus.africacode.net/topics/js-and-node-specific/expressjs/ be upgraded to cover this? 

Remember if this is actually needed as a workshop we can leave it in but the better outcome is that the topic is upgraded to cover this

File: content/topics/general-coding-best-practices/_index.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+---
+_db_id: 19
+content_type: topic
+ready: true
+title: Clean Code (language agnostic)
```
Comment (ry-oc) [inline content/topics/general-coding-best-practices/_index.md]: There is a comment on one of the syllabusses about this

File: content/topics/git-basics/_index.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
+content_type: topic
+ready: true
+title: Git Basics
+---
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vTfYzPpWfCU3ZD9EPT9PhH-7bDC45fQUz1_YkTJfB80eV1jzAiJ2M55hKJqk59d9hV_k0e6TxO_SLQu/embed?start=false&loop=false&delayms=60000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```
Comment (ry-oc) [inline content/topics/git-basics/_index.md]: Does topics/git/_index.md cover this info or does it need to be updated? once topics/git/_index.md is updated this be depreciated

File: content/topics/intro-to-css/_index.md (hunk 2 above)
Comment (ry-oc) [inline content/topics/intro-to-css/_index.md]: This topic already exists and is added to the recruits backlog so this can be depreciated

**Review by ry-oc — CHANGES_REQUESTED** [review]
Quite a few changes but so far so good.

Apologies for the many comments but I have been having technical issues :(

**Comment by ry-oc** [comment]
Please can you resolve merge conflicts on this then I can review the whole thing from scratch :)

File: content/syllabuses/data-sci.md [hunk 14]
```diff
@@ -9,14 +9,14 @@ _db_id: 2
 - {{% contentlink path="projects/katas/level-2" flavour="python" %}}
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python" %}}
 - {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
```
Comment (sheenarbw) [inline content/syllabuses/data-sci.md]: let's keep general-coding best practices. And then use the `topics/clean-code/python` as well

Comment (sheenarbw) [reply content/syllabuses/data-sci.md]: The  ` general-coding best practices` presentation has some good stuff in there. I might turn it into something bigger at some point. But I'm not certain that those concepts are represented in the language specific clean-code goodies

File: content/syllabuses/java-part-1.md [hunk 15]
```diff
@@ -5,17 +5,17 @@ _db_id: 7
 
 - {{% contentlink path="agile/agile-scrum" %}}
 - {{% contentlink path="topics/solo-learn/java/basic-concepts" %}}
-- {{% contentlink path="workshops/big-picture-web-dev" %}}
+- {{% contentlink path="topics/big-picture-web-dev" %}}
 - {{% contentlink path="topics/solo-learn/java/conditionals-and-loops" %}}
 - {{% contentlink path="topics/solo-learn/java/arrays" %}}
+- {{% contentlink path="topics/clean-code/java" %}}
 - {{% contentlink path="topics/clean-code" %}}
```
Comment (sheenarbw) [inline content/syllabuses/java-part-1.md]: can probably remove - {{% contentlink path="topics/clean-code" %}}
since - {{% contentlink path="topics/clean-code/java" %}} is here

File: content/syllabuses/java-part-2.md [hunk 16]
```diff
@@ -3,7 +3,7 @@ _db_id: 57
 title: Java Systems Development - part 2
 ---
 
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/clean-code" %}}
```
Comment (sheenarbw) [inline content/syllabuses/java-part-2.md]: Why is this mentioned in part 1 and part 2? - {{% contentlink path="topics/clean-code" %}}

Comment (sheenarbw) [reply content/syllabuses/java-part-2.md]: why remove general-coding-best practices?

File: content/syllabuses/web-dev-part-2.md [hunk 17]
```diff
@@ -4,7 +4,7 @@ title: Web Development - part 2
 ---
 
 - {{% contentlink path="topics/web-frontend/intro-to-web-design" %}}
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
```
Comment (sheenarbw) [inline content/syllabuses/web-dev-part-2.md]: dont remove - {{% contentlink path="workshops/general-coding-best-practices" %}}

File: content/syllabuses/web-dev-part-2.md [hunk 18]
```diff
@@ -4,7 +4,7 @@ title: Web Development - part 2
...
+- {{% contentlink path="topics/clean-code" %}}
```
Comment (sheenarbw) [inline content/syllabuses/web-dev-part-2.md]: is this meant to be clean code/javascript?
(+ just check that it isn't already mentioned in part 1)

File: content/syllabuses/web-frontend-foundations.md [hunk 19]
```diff
@@ -11,20 +11,19 @@ title: Web frontend foundations
...
 - {{% contentlink path="projects/katas/level-2" flavour="javascript" %}}
 - {{% contentlink path="agile/agile-scrum" %}}
-- {{% contentlink path="workshops/big-picture-web-dev" %}}
+- {{% contentlink path="topics/big-picture-web-dev" %}}
 - {{% contentlink path="topics/js-and-node-specific/super-basic-intro-to-node" %}}
 - {{% contentlink path="topics/web-frontend/intro-to-web-design/" %}}
 - {{% contentlink path="projects/first-personal-website" %}}
 - {{% contentlink path="topics/free-code-camp/responsive-web-design/3-applied-visual-design" %}}
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/clean-code/" %}}
```
Comment (sheenarbw) [inline content/syllabuses/web-frontend-foundations.md]: same comments

File: content/syllabuses/web-frontend-foundations.md [hunk 20]
```diff
@@ -11,20 +11,19 @@ title: Web frontend foundations
...
 - {{% contentlink path="topics/jasmine-unit-tests" %}}
 - {{% contentlink path="topics/free-code-camp/javascript-data-structures-and-algorithms/5-basic-data-structures" %}}
 - {{% contentlink path="topics/free-code-camp/javascript-data-structures-and-algorithms/6-basic-algorithmic-scripting" %}}
 - {{% contentlink path="topics/free-code-camp/javascript-data-structures-and-algorithms/7-object-orientated-programming" %}}
 - {{% contentlink path="projects/oop/animals/part2"  flavour="javascript" %}}
 - {{% contentlink path="topics/free-code-camp/responsive-web-design/4-applied-accessability" %}}
 - {{% contentlink path="topics/free-code-camp/responsive-web-design/5-responsive-web-design-principles" %}}
-
```
Comment (sheenarbw) [inline content/syllabuses/web-frontend-foundations.md:27]: empty line?

File: content/topics/data-science-specific/google-data-studio/_index.md [hunk 21]
```diff
@@ -9,8 +9,6 @@ title: Google Data Studio
 
 Dashboarding is easy with Google Data Studio. Google Data Studio helps you import, manipulate and display your data in an easy to understand way. If you know Google Data Studio, you'll also find other popular data analytics tools such as Power BI (Microsoft), Tableau and Qlikview easy to learn.
 
-The slides for the data visualisation / business intelligence workshop can be found in: {{% contentlink path="workshops/business-intelligence-tools" %}}
```
Comment (sheenarbw) [inline content/topics/data-science-specific/google-data-studio/_index.md:12]: Dont we need to tell people were to find the slides?

File: content/workshops/business-intelligence-tools/_index.md [hunk 22]
```diff
@@ -1,8 +0,0 @@
----
-_db_id: 13
-content_type: workshop
-ready: true
-title: Business intelligence tools
----
-
```
Comment (sheenarbw) [inline content/workshops/business-intelligence-tools/_index.md]: why was this deleted?

Comment (sheenarbw) [reply content/workshops/business-intelligence-tools/_index.md]: There is definately information in the slides. 

Waere our data science peeps consulted? I think this is referenced in their syllabus. But they are pretty junior so I'm not sure that they should be culling parts of the syllabus anyway

File: content/workshops/clean-code/_index.md [hunk 23]
```diff
@@ -1,9 +0,0 @@
----
-_db_id: 14
-content_type: workshop
-title: Clean Code
----
-
-> “_Any fool can write code that a computer can understand.
-> Good programmers write code that humans can understand_.”
```
Comment (sheenarbw) [inline content/workshops/clean-code/_index.md]: Rathre leave this in. That way this shows up in the contents in the side panel.

File: content/workshops/intro-to-css-architecture/_index.md [hunk 24]
```diff
@@ -1,16 +0,0 @@
----
-_db_id: 7
-content_type: workshop
-ready: true
-title: Intro to CSS architecture
----
-
-The followings are to be demonstrated and addressed during the workshop
```
Comment (sheenarbw) [inline content/workshops/intro-to-css-architecture/_index.md]: dont delete this

Comment (sheenarbw) [reply content/workshops/intro-to-css-architecture/_index.md]: unless the topic covers everything it should?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
ok. It's pretty close. But fn weird. This task should hav been much simpler

File: content/syllabuses/web-frontend-foundations.md (hunk 20 above)
Comment (ry-oc) [reply content/syllabuses/web-frontend-foundations.md:27]: Yeah random empty line was removed

File: content/topics/data-science-specific/google-data-studio/_index.md (hunk 21 above)
Comment (ry-oc) [reply content/topics/data-science-specific/google-data-studio/_index.md:12]: Fixed

**Comment by ry-oc** [comment]
I can no longer push changes here so have to close this and open a new pr
